### PR TITLE
Adding status of Arcade Validation and Dependency Flow

### PR DESCRIPTION
**Pull request by missymessa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ Arcade is intended to provide well-understood and consistent mechanisms for cons
 
 Azure DevOps [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/arcade/arcade-ci)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=208)
 
+## Validation & Dependency Flow Status
+
+### Current Version of Arcade in `.NET Eng - Latest`
+
+`Microsoft.DotNet.Arcade.Sdk @ 5.0.0-beta.20201.2`
```
Comment (riarenas) [inline README.md:15]: We already provide this information in barviz without needing users to run a darc command, can we point them to those links?

Comment (missymessa) [reply README.md:15]: If we already have it in BARViz, I would much prefer that!

Comment (riarenas) [reply README.md:15]: Yep, https://github.com/dotnet/arcade/issues/5230#issuecomment-611709298

Comment (missymessa) [reply README.md:15]: Cool, found them, I'll update these sections with links to Arcade in the Latest and Validation channels. I'll leave the status stuff in there, though.

**Pull request merged**